src/Api/AbstractOystApiClient.php

```diff
@@ -8,36 +13,21 @@
  * @license  Copyright 2017, Oyst
  * @link     http://www.oyst.com
  */
-namespace Oyst\Api;
-
-use Guzzle\Http\Exception\ClientErrorResponseException;
-use Guzzle\Service\Client;
-
 abstract class AbstractOystApiClient
 {
-    /**
-     * @var Client
-     */
+    /** @var Client */
```

Comment:
Multiline is better to describe each properties and will be useful for documentation generation, same fo all properties bellow.